themes/plugins.jquery.com/style.css

```diff
@@ -108,7 +108,7 @@ a {
 .github-activity .info-block .number {
 	border-top-left-radius: 4px;
 	border-top-right-radius: 4px;
-	background: black url('images/dark-grey-tile.png');
+	background: black url('/jquery-wp-content/themes/jquery/images/dark-grey-tile.png');
```

Comment:
I'd recommend `../jquery/images` instead so as to not hardcode production installation paths. In particular, on the new server setup (with standalone WordPress sites instead of a single multi-site configuration), these paths are different.

For example, on https://stage.learn.jquery.com, `/jquery-wp-content` is a 404, and instead the WordPress canonical standard `/wp-content` is availalbe only, with individual themes linked from an internal checkout of this repository.

Reply:
Good suggestion. I've change all the dark-grey-tile references. I've kept the commit subject as plugins because jquery.org will soon be gone.